### DISCUSSION TITLE
invalid active developer path, missing xcrun

### DISCUSSION THREAD

**Discussion by ASKapat** [idaholab/moose#17482] [Q&A General] [answered]
Hi, 

After updating my MacOS to 11.2.3 from 11.2 (before which I wasn't having this issue) I'm now getting the following error when I activate moose:

xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

**Comment by cticenhour** [comment, marked as answer]
First I would try re-installing CommandLineTools. In your Terminal, run: 

```
xcode-select --install
```

Note that after installation completes, you should probably close your Terminal window and open a new one.

**Reply by ASKapat** [reply]
That did it, thanks for the help!

**Reply by cticenhour** [reply]
Glad I could help!

**Reply by Trevor1910** [reply]
Perfect fix!  Thanks a lot!
